.changeset/many-masks-smoke.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/repo-tools': minor
+'@backstage/cli': minor
+---
+
+Remove support for the deprecated `--experimental-type-build` option for `package build`.
```

Comment:
We should remember to call this out in the release notes, almost worth a BREAKING here, but can also see the point in not bothering

Reply:
Yep already marked as experimental through it's name and `[DEPRECATED]` in docs. But indeed, I created the 1.20 notes doc and added an entry for this